<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inbox</class>
 <widget class="QDialog" name="Inbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="hopThuDenPage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>370</width>
          <height>540</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Hộp thư đến</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="capNhatButton">
               <property name="text">
                <string>Cập nhật</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableView" name="hopThuDenTable">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="hopThuDiPage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>418</width>
          <height>540</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Thư đã gửi</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTableView" name="hopThuDiTable">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="soanTinNhanPage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>418</width>
          <height>540</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Soạn tin nhắn</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="tuaDe">
             <property name="placeholderText">
              <string>Tựa đề</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="nguoiNhan">
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="insertPolicy">
              <enum>QComboBox::NoInsert</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="loiNhan">
             <property name="placeholderText">
              <string>Lời nhắn</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="guiButton">
             <property name="text">
              <string>Gửi</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="hienThiTinNhan">
     <layout class="QVBoxLayout" name="xemTinNhan">
      <item>
       <widget class="QLineEdit" name="tieuDeLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Tiêu đề</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nguoiGuiLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Người gửi/Người nhận</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="thoiGianLineEdit">
        <property name="placeholderText">
         <string>Thời gian</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="loiNhanTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Lời nhắn</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tuaDe</tabstop>
  <tabstop>nguoiNhan</tabstop>
  <tabstop>loiNhan</tabstop>
  <tabstop>guiButton</tabstop>
  <tabstop>hopThuDenTable</tabstop>
  <tabstop>loiNhanTextEdit</tabstop>
  <tabstop>hopThuDiTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
